operator/src/main/java/oracle/kubernetes/operator/helpers/PodStepContext.java

```diff
@@ -535,7 +551,9 @@ protected V1ObjectMeta createMetadata() {
           .ifPresent(hash -> addHashLabel(metadata, LabelConstants.MODEL_IN_IMAGE_MODEL_SECRETS_HASH, hash));
 
     // Add prometheus annotations. This will overwrite any custom annotations with same name.
-    AnnotationHelper.annotateForPrometheus(metadata, getDefaultPort());
+    // Prometheus does not support "prometheus.io/scheme".  The scheme(http/https) can be set
+    // in the Prometheus Chart values yaml under the "extraScrapeConfigs:" section.
```

Comment:
Do we need to make changes elsewhere to accommodate these `extraScrapeConfigs`? For instance, are there documentation or blogs that must now be updated?

Reply:
@rjeberhard , I think we do for the use case when only the SSL port of the managed server is enabled.  The other use case I found that breaks the monitoring exporter app is when the managed server has the non-SSL port enabled + the administration port enabled.  Should I enter a tracking bug or JIRA for it?